tests/unit/aiplatform/test_training_jobs.py

```diff
@@ -73,7 +74,8 @@
 _TEST_GCS_PATH = f"{_TEST_BUCKET_NAME}/{_TEST_GCS_PATH_WITHOUT_BUCKET}"
 _TEST_GCS_PATH_WITH_TRAILING_SLASH = f"{_TEST_GCS_PATH}/"
 _TEST_LOCAL_SCRIPT_FILE_NAME = "____test____script.py"
-_TEST_LOCAL_SCRIPT_FILE_PATH = f"path/to/{_TEST_LOCAL_SCRIPT_FILE_NAME}"
+_TEST_TEMPDIR = tempfile.mkdtemp()
+_TEST_LOCAL_SCRIPT_FILE_PATH = os.path.join(_TEST_TEMPDIR, _TEST_LOCAL_SCRIPT_FILE_NAME)
```

Comment:
Do you think you can afford dependency on `absl-py`? If, standardizing on tmp dir handling, with ability to inject via a public envvar, is benefitial.

https://github.com/abseil/abseil-py/blob/main/absl/testing/absltest.py#L158-L165


Alternatively, `pytest` also has a built in fixture to do so in a standard way, but I understand that `pytest` is used only for test driver, but not as a unittest utility framework as it is now, so it will be anyways additional dependency.


https://docs.pytest.org/en/6.2.x/tmpdir.html#base-temporary-directory